src/PerimeterxFirstPartyClient.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace Perimeterx;
+
+final class PerimeterxFirstPartyClient {
+    const CLIENT_DOMAIN = 'client.perimeterx.net';
```

Comment:
**General question that is not blocking this PR or anything:**

Why don't we use 1 backend for all first_party request as in Fastly? I think this should be the default implementation, what do you think?

Reply:
Remember we found a 1-backend issue in the portal? It takes additional setup and could result in more issues so I'd rather keep with the implementation of the node-core enforcer if there's no downside to having multiple backends.

Reply:
I think personally it is better to have 1 backend as a best practice if we will use it to ALL Enforcers without additional setup as this will be the default, but this of course not in the scope of this task just your say :)